firmware/README.md

```diff
@@ -26,25 +26,70 @@
 ### Arduino IDE
 For simplicity, Arduino IDE has been used in the documentation but other IDEs can also be used. Arduino IDE, can be downloaded from [Arduino's website](https://www.arduino.cc/en/software). The Arduino IDE is used for uploading the firmware to the microcontroller.
 
-### Flood Sensor Library
+Add HelTec Boards to the IDE, there are two ways:
+ - Using Arduino's Boards Manager.
+     - Open Arduino IDE, and goto File->Preferences->Settings
+      <br> 
+      <img src="img/IDE preferences.png" width="440" height="300"> 
+     
+     - Enter the [URL](https://github.com/HelTecAutomation/CubeCell-Arduino/releases/download/V1.5.0/package_CubeCell_index.json) to board manager URLs. Then hit 		 ok.
+     - After that goto Tools->Board:->Boards Manager, search CubeCell in the new pop-up dialog, select the latest releases and click install. 
+      <br>
+      <img src="img/IDE.png" width="900" height="200">
+ - Using Github.
+     - Follow the instructions mentioned [here.](https://github.com/HelTecAutomation/CubeCell-Arduino/blob/master/InstallGuide/windows.md)
+     </a>
 Download the Flood Sensor Library from [here](https://github.com/floodnet-nyc/flood-sensor). Following are the steps to install this library:
-
 1. Find Arduino libraries folder
-2. Clone the library and copy the `/firmware` folder into the Arduino>libraries
+2. Copy the `firmware` folder into the Arduino>libraries
 
 ### Installing CubeCell Library
 This repo is built to run on CubeCell's example library. To install the CubeCell's library, follow the instructions on their repo [here](https://github.com/HelTecAutomation/CubeCell-Arduino/tree/master/InstallGuide)
 
 ### Registering the device to The Things Network
-
+ Setting up The Things Network
+  - Make an account on The Things Network at [here](https://www.thethingsnetwork.org/)
```

Comment:
Add a step here to demonstrate logging into the TTN console and provide a link to the console `https://nam1.cloud.thethings.network/console/`